bin/si/src/version.txt

```diff
@@ -1 +1 @@
-20230731231120
\ No newline at end of file
+20190703.164251.0-sha.021b4563a
```

Comment:
I made this update for 2 reasons:

1. The existing version string was old and did not correspond to our current version string format
2. The updated version string corresponds to a real commit 😉